src/layoutPicker.ts

```diff
@@ -24,11 +25,31 @@ const layouts: QuickPickItem[] = [
   ),
 ];
 
+const layoutTala = new LayoutItem("tala", "Terrastruct's AutoLayout Approach");
+
+const talaPluginName: string =
+  process.platform === "win32" ? "d2plugin-tala.dll" : "d2plugin-tala";
```

Comment:
@nhooyr what's the path for windows tala? i know it's not dll, is it just exe?

Reply:
`d2plugin-tala.exe` should work.

Reply:
@BarryNolte can you change to that please?